core-web/apps/dotcms-ui/src/app/api/services/dot-templates/dot-templates.service.ts

```diff
@@ -85,19 +86,34 @@ export class DotTemplatesService {
     }
 
     /**
-     * Save and Publish a template
+     * Save and Publish a template Old version, use saveAndPublish instead
      * @param {DotTemplate} values
      * @returns Observable<DotTemplate>
      * @memberof DotTemplatesService
+     * @deprecated
      */
-    saveAndPublish(values: DotTemplate): Observable<DotTemplate> {
+    saveAndPublishOld(values: DotTemplate): Observable<DotTemplate> {
```

Comment:
why keep if not used ?

Reply:
Yeah, maybe we can delete this method at all